Library/Homebrew/cmd/deps.rb

```diff
@@ -22,9 +22,9 @@ def deps_args
         may be appended to the command. When given multiple formula arguments,
         show the intersection of dependencies for each formula.
       EOS
-      switch "-n",
+      switch "--topological", "-n",
```

Comment:
```suggestion
      switch "-n", "--topological",
```

Reply:
@EricFromCanada Why the reversal? Feels normal to have the longer flag first. Thoughts?

Reply:
Mainly because every other command lists the short flag first, and this also matches the manpage and `brew -h` output.

Reply:
Whoops, this was the opposite to what I thought then!